npm/webpack-dev-server/src/makeWebpackConfig.ts

```diff
@@ -1,28 +1,51 @@
 import { debug as debugFn } from 'debug'
 import * as path from 'path'
-import { Configuration } from 'webpack'
+import * as webpack from 'webpack'
 import { merge } from 'webpack-merge'
+import defaultWebpackConfig from './webpack.config'
+import LazyCompilePlugin from 'lazy-compile-webpack-plugin'
```

Comment:
This is the one: https://github.com/liximomo/lazy-compile-webpack-plugin ?

I'm surprised more people are not using this 🤔 seems not well known